includes/User.php

```diff
@@ -2648,22 +2648,7 @@ public function getGlobalAttribute( $attribute, $default = null ) {
 	 * @return bool
 	 */
 	private function shouldGetAttributeFromService( $attributeName ) {
-		global $wgEnableReadsFromAttributeService;
-
-		if ( empty( $wgEnableReadsFromAttributeService ) ) {
-			return false;
-		}
-
-		// User is anonymous or nonexistant
-		if ( $this->getId() == 0 ) {
-			return false;
```

Comment:
We now do this check inside of `UserAttributes.php` [here](https://github.com/Wikia/app/pull/10875/files#diff-3fe2c8a946b8da62e3fdb9ba3f5ff012R71). By returning an empty array there, we just used default values for attributes. See the logic [here](https://github.com/Wikia/app/blob/dev/lib/Wikia/src/Service/User/Attributes/UserAttributes.php#L56-L66).